### PR TITLE
AnneJu step 12: Add due date of the task

### PR DESCRIPTION
**Pull request by anneju**
#### Description
- as title

#### Stories
- [x] Allow create, update, show due_day on app
- [x] prevent due_day be empty by adding validation
- [x] Write Specs

#### Related RSpecs
```
rspec spec/models/tasks_spec.rb:9 spec/system/tasks_spec.rb:66:105:122:144:182
```

### CODE REVIEW

File: myapp/app/views/tasks/index.html.erb [hunk 1]
```diff
@@ -8,6 +8,7 @@
       <td><%= I18n.t('activerecord.attributes.task.id') %></td>
       <td><%= I18n.t('activerecord.attributes.task.title') %></td>
       <td><%= I18n.t('activerecord.attributes.task.description') %></td>
+      <td><%= I18n.t('activerecord.attributes.task.due_date') %></td>
```
Comment (leesanghun6310) [inline myapp/app/views/tasks/index.html.erb]: I seem to be good using loops than this code

**Comment by anneju** [comment]
Hi @leesanghun6310 I have updated 1d0efde8880aec551e51043fc76a40a753220e7a based on your feedback~ Please help take a look again~

**Review by leesanghun6310 — APPROVED** [review]
LGTM

**Comment by leesanghun6310** [comment]
完璧に理解した

**Pull request merged**